torchci/rockset/metrics/__sql/last_successful_jobs.sql

```diff
@@ -12,7 +12,7 @@ with successful_jobs as (
         JOIN workflow_run workflow on workflow.id = job.run_id
     where
         workflow.repository.full_name = 'pytorch/pytorch'
-        AND workflow.head_branch = 'master'
```

Comment:
fd8bc3a2afc989a2 for this one

Reply:
@janeyx99 This one didn't seem to work.

Reply:
Oh, this is because I had added a parameter to it as well. Could you edit the corresponding lambda.json file to add the parameter jobNames? (Shown in the test failure).